src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -210,7 +210,7 @@ public boolean runUntilEnd() throws IOException, InterruptedException {
             flags.add("-H:CustomLD=" + ldlld.toAbsolutePath().toString());
         }
         if (projectConfiguration.getGraalVersion().compareTo(new Version("20.1.0")) > 0) {
-            flags.add("-H:+UseBionicC");
+            flags.add("--libc=bionic");
```

Comment:
since what version is this? If it is 20.3-dev and on, we can test for that.

Reply:
I believe it started with one of the last versions of 20.2.0-dev (I tested https://github.com/graalvm/graalvm-ce-dev-builds/releases/tag/20.2.0-dev-20200718_0205). 
It's safe to check for version 20.3.0 though.

Reply:
@jperedadnr You are correct, however, this flag will be included in 20.2 too, so if you are running on 20.2, it should be `--libc=bionic`

Reply:
Yes, I reported that in https://github.com/gluonhq/client-maven-plugin/issues/211 for 20.2.0-dev-20200718_0205. Therefore it is currently not possible to build for Android.

Reply:
I've pushed a change to check version 20.2 and above. This will allow using the latest 20.2.0-dev version and the official release 20.2 (when released), or any 20.3-dev versions and beyond.